internal/admin/dashboard/templates/index.html

```
@@ -612,8 +612,297 @@ <h3 x-text="aliasFormMode === 'edit' ? (aliasForm.name || aliasFormOriginalName)
+    <div class="alert alert-warning" x-show="authError">
+        Authentication required. Enter your API key in the sidebar to view data.
+    </div>
+    <div class="alert alert-warning" x-show="!executionPlansAvailable && !authError">
+        Workflows feature is unavailable.
+    </div>
+    <div class="alert alert-warning" x-show="executionPlanError && !authError" x-text="executionPlanError"></div>
+    <div class="alert alert-success" x-show="executionPlanNotice" x-text="executionPlanNotice"></div>
+
+    <div class="table-toolbar" x-show="executionPlansAvailable">
+        <div class="table-toolbar-main">
+            <input type="text" placeholder="Filter by scope, name, hash, or guardrail..." x-model="executionPlanFilter" class="filter-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)